src/main/java/org/jabref/gui/preferences/FileTab.fxml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.CheckBox?>
-<?import javafx.scene.control.ComboBox?>
 <?import javafx.scene.control.Label?>
-<?import javafx.scene.control.RadioButton?>
-<?import javafx.scene.control.TextField?>
-<?import javafx.scene.control.ToggleGroup?>
-<?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.VBox?>
 <?import org.jabref.gui.commonfxcontrols.SaveOrderConfigPanel?>
+<?import javafx.scene.control.ToggleGroup?>
```

Comment:
Is there any system in reordering the imports in the fxmls? If not, why not leave it alphabetically like every other import in the .Java files...

Reply:
Got it!